ui/src/message_popup/author/question_editing_components/editing_component.jsx

```diff
@@ -0,0 +1,14 @@
+/* @flow weak */
+import QuestionText from './question_text.jsx';
+import Students from './students.jsx';
+import Examples from './examples.jsx';
+import LearningObjectives from './learning_objectives.jsx';
+import Indicators from './indicators.jsx';
+
+export {
```

Comment:
I imagine you're following the pattern in the Message PopUp page, which is a good strategy!  But this doesn't seem particularly useful to me, you can just import these directly.

The file exporting specific pieces in `message_popup/index.js` is because I was trying to be explicit about what the entry points were into the MessagePopUp UI from other parts of the product (e.g, viewing scores across all products, navigating from the shared navigation drawer).  As we were talking about in-person, I haven't done enough to talk about this or make this clear enough.  But you don't need to worry about that here, and can just import these editing components directly in the question page.